.eslintrc.js

```diff
@@ -1,11 +1,14 @@
 module.exports = {
   extends: ['algolia', 'algolia/jest', 'algolia/react', 'algolia/typescript'],
+  plugins: ['react-hooks'],
```

Comment:
can you open an issue or PR to add this to our react config?

Reply:
Will open a PR!

Reply:
algolia/eslint-config-algolia#118